File: nova-api/src/main/java/xyz/xenondevs/nova/api/event/NovaLoadDataEvent.java
package xyz.xenondevs.nova.api.event;

import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;
import org.jetbrains.annotations.NotNull;

public class NovaLoadDataEvent extends Event {
    
    private static final @NotNull HandlerList HANDLERS = new HandlerList();
    
    public NovaLoadDataEvent() {
        super(true);
    }
    
    public static @NotNull HandlerList getHandlerList() {
        return HANDLERS;
    }
    
    @Override
    public @NotNull HandlerList getHandlers() {
        return HANDLERS;
    }
    
}

File: Nova-API/src/main/kotlin/xyz/xenondevs/nova/api/event/NovaLoadDataEvent.kt
package xyz.xenondevs.nova.api.event

import org.bukkit.event.Event
import org.bukkit.event.HandlerList

class NovaLoadDataEvent : Event(true) {
    
    companion object {
        @JvmStatic
        private val HANDLERS = HandlerList()
        
        @JvmStatic
        fun getHandlerList(): HandlerList {
            return HANDLERS
        }
        
    }
    
    override fun getHandlers(): HandlerList {
        return HANDLERS
    }
    
}